packages/frontend/app/components/HeaderBar.tsx

```diff
@@ -340,7 +343,12 @@ const NavBar = ({
               {checkCourseCreatePermissions(userInfo, organizationSettings) && (
                 <NavigationMenuItem>
                   <Link
-                    className="md:pl-8"
+                    className={cn(
+                      '!md:pl-8',
```

Comment:
```suggestion
                      'md:pl-8',
```
The other links don't have the `!important`, so I'm assuming adding it was a mistake